content/release-10-17-0/applications-10-17-0.md

```diff
@@ -153,6 +175,27 @@ Fix Version</th>
 <td>10.16.102.0</td>
 </tr>
 
+<tr>
+<td>Web SDK</td>
+<td>Opening the widget configuration for a microfrontend plugin multiple times does no longer cause performance issues in the UI.</td>
+<td>MTM-50306</td>
+<td>10.17.0.24</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>The <code>DynamicComponentService</code> now waits for all plugins to be loaded before emitting the first value via its <code>items$</code> observable.</td>
+<td>MTM-50985</td>
+<td>10.17.0.5</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>A toggle has been added to the plugin installation dialog, which allows to filter the plugin versions for only the latest versions or all versions.</td>

```

Comment:
is it "latest versions" or "latest version"? if plural, how is "latest" then defined?

Reply:
Should be latest version (sing.), will change it